docs/slack-apps/slack-intro.md

```diff
@@ -0,0 +1,46 @@
+---
+id: slack-intro
+title: Your First Slack App
+sidebar_label: Your First Slack App
+---
+
+Building Slack apps in Dark is a quick and easy process. This page will cover the basics, but more specific examples are covered in the tutorials.
+
+## Slack Configuration
+
+Start by creating an app in Slack. You can do so quickly by clicking [here](https://api.slack.com/apps?new_app=1). Note that you will need to have the ability to add apps to at least one Slack group in order to create an app.
```

Comment:
General rule for docs on the internet: links should never say "here", but should describe what's behind it.  In this case:

```
Start by [creating an app in Slack](https://api.slack.com/apps?new_app=1).
```